packages/workflow/src/Interfaces.ts

```diff
@@ -1643,6 +1643,13 @@ export interface INodeType {
 			[method in WebhookSetupMethodNames]: (this: IHookFunctions) => Promise<boolean>;
 		};
 	};
+	nonRoutingOperations?: {
```

Comment:
Naming-wise I prefer to avoid negatives, plus they're still inside the routing node. What about `customOperations` with a docline on top explaining what this is?